modules/planning/on_lane_planning.cc

```diff
@@ -211,7 +211,9 @@ void OnLanePlanning::RunOnce(const LocalView& local_view,
   VehicleState vehicle_state =
       VehicleStateProvider::Instance()->vehicle_state();
   const double vehicle_state_timestamp = vehicle_state.timestamp();
-  DCHECK_GE(start_timestamp, vehicle_state_timestamp);
+  DCHECK_GE(start_timestamp, vehicle_state_timestamp)
+      << "start_time_stamp is behind vehicle_state_timestamp by"
+      << start_time_stamp - vehicle_state_timestamp << "secs";
```

Comment:
Leave some space before and after numbers.

Reply:
Done